Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -831,6 +831,8 @@ extension BrowserViewController: TabManagerDelegate {
         toolbar?.updateBackStatus(selected?.canGoBack ?? false)
         toolbar?.updateFowardStatus(selected?.canGoForward ?? false)
         toolbar?.updateReloadStatus(selected?.webView.loading ?? false)
+        toolbar?.updatePageStatus(isWebPage(selected!.displayURL!))
```

Comment:
Both `selected` and `displayURL` can be `nil` in certain cases. `selected` can be `nil` if the user closes all tabs; `displayURL` can be `nil` if the webview discards its content (which can happen if the OS is running low on memory). So these should be changed from `!` to `?`.